react_ws_src/src/views/ttt/GameMain.js

```diff
@@ -1,11 +1,10 @@
-import React, {Component} from 'react'
```

Comment:
Sorry about the re-formatting: when I added the new `get_computer_move` function into this file, my IDE automatically cleaned up the imports.